sdk/file_sdk/Common/file_format.cpp

```diff
@@ -0,0 +1,25 @@
+#include "file_format.h"
+
+namespace mip {
+namespace file {
+
+FileFormat::FileFormat(
+    std::shared_ptr<IStream> file,
+    const string& extension)
+  : mFile(file),
+    mExtension(extension)
+{
+}
+
+const vector<mip::Tag>& mip::file::FileFormat::GetTags()
+{
+  return mTags;

```

Comment:
i know, but why do we need another function for that? currently, we read the tags in the ctor and get tags returns it

---
In reply to: [134443149](https://github.com/AzureAD/rms-sdk-for-cpp/pull/133#discussion_r134443149) [](ancestors = 134443149,134442532,134435478)